gpii/node_modules/accessRequester/test/AccessRequesterTests.js

```diff
@@ -41,12 +41,19 @@ gpii.tests.accessRequester.setupNock = function (config) {
         .reply(config.status, config.response);
 };
 
+gpii.tests.accessRequester.cleanNock = function () {
+    nock.cleanAll();
+};
+
 // The customized accessRequester component for tests
 fluid.defaults("gpii.tests.accessRequester", {
     gradeNames: ["gpii.accessRequester"],
     gpiiToken: gpii.tests.accessRequester.gpiiToken,
     url: gpii.tests.accessRequester.hostname + gpii.tests.accessRequester.path,
-    clientCredentialDataSourceGrade: "gpii.accessRequester.clientCredentialDataSource.file"
+    clientCredentialDataSourceGrade: "gpii.accessRequester.clientCredentialDataSource.file",
+    listeners: {
+        "onDestroy.cleanNock": "gpii.tests.accessRequester.cleanNock"
```

Comment:
Currently the setup of Nock expectations is done at the start of a test sequence that requires HTTP request mock support. With this change, we are cleaning up Nock at the accessRequestor onDestroy() lifecycle event. It would be better to have the setup and cleanup be managed by the same component or mechanism so that the relationship is clear and that they are reliably managed together.

I would either do the setup and cleanup both as test sequence actions, or have the setup and cleanup be managed as lifecycle actions on the same component.